.github/workflows/build_ui.yml

```diff
@@ -49,6 +49,13 @@ jobs:
           cat .npmrc
           cat .yarnrc
 
+      - name: Setting up .env
+        shell: bash
+        run: |
+          ENV_JSON='${{secrets.TESSELL_UI_ENV_SECRET}}'
+          echo "$ENV_JSON" | jq -r 'to_entries | .[] | "\(.key)=\"\(.value)\"" ' > .env
```

Comment:
This will overwrite the existing `.env`. We need to append to existing `.env`

Reply:
but there's no existing .env

Reply:
@varenyatessell check [this PR](https://github.com/TessellDevelopment/tessell-ui/pull/2404), it will be merged together with yours